src/unpackertool/plugins/upx/decompressors/decompressor_nrv.h

```diff
@@ -9,6 +9,8 @@
 #include "unpackertool/plugins/upx/decompressors/decompressor.h"
 #include "retdec/unpacker/decompression/nrv/bit_parsers.h"
 
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.